core/src/main/scala/org/apache/spark/scheduler/cluster/StandaloneSchedulerBackend.scala

```diff
@@ -231,6 +232,20 @@ private[spark] class StandaloneSchedulerBackend(
     }
   }
 
+  private def extractDriverLogUrls(): Map[String, String] = {
```

Comment:
Shall we inline this function to the `getDriverLogUrls`?

Reply:
ok, done